docs/faq.md

```diff
@@ -74,18 +84,25 @@ open class Element {
  *  GREYAssertTrue(element.text != "", reason: "get text failed")
  */
 public func grey_getText(_ elementCopy: Element) -> GREYActionBlock {
-  return GREYActionBlock.action(withName: "get text", 
-  constraints: grey_respondsToSelector(#selector(getter: UILabel.text))) { element, 
+  return GREYActionBlock.action(withName: "get text",
+  constraints: grey_respondsToSelector(#selector(getter: UILabel.text))) { element,
                                                                            errorOrNil -> Bool in
         let elementObject = element as? NSObject
-        let text = elementObject?.perform(#selector(getter: UILabel.text), 
+        let text = elementObject?.perform(#selector(getter: UILabel.text),
                                           with: nil)?.takeRetainedValue() as? String
         elementCopy.text = text ?? ""
         return true
     }
 }
 ```
 
+**I get a crash with “Could not swizzle …”**
+
+This means that EarlGrey is trying to swizzle a method that it has swizzled before. It is a result
+of EarlGrey being linked to more than once. Ensure that only the **Test Target**
+depends on *EarlGrey.framework* and EarlGrey.framework is embedded in the app under test (i.e. *$TEST_HOST*) from the
+test target's build phase.
+
 **How do I check whether an element exists in the UI hierarchy?**
```

Comment:
Under **How should I handle animations?**, there's a code snippet that shows how to speed up animations:
`UIApplication.sharedApplication().keyWindow?.layer.speed = 100`

Could we change it to use the TestHelper instead?
https://github.com/google/EarlGrey/blob/master/EarlGrey/Common/GREYTestHelper.m